### ISSUE TITLE
Muted WhatsApp chats update the badge as if they were not muted

### ISSUE THREAD

**Issue by Athurios** [TheGoddessInari/hamsket#130]
### Steps to reproduce

1. Add WhatsApp as a service using the default settings
2. Have a chat on mute
3. Receive a notification from the muted chat

### Expected behavior

In the Hamsket services tab and the Windows taskbar, the badges only update on a message from a non-muted chat. 

### Actual behavior

Muted chats update the WhatsApp badge inside Hamsket and the badge in the taskbar, ignoring the mute. The tray icon also turns red regardless of the mute (I am not sure if this is also unintended though).

### ENVIRONMENT

> Hamsket: v0.5.19

> OS: Windows 10

> Arch: x64

> BuildVersion: e630f2a

> Does this happen upstream: No

**Comment by TheGoddessInari** [comment]
Try the Windows build from here, see if that helps.

https://dev.azure.com/TheGoddessInari/hamsket/_build/results?buildId=149&view=artifacts

**Comment by Athurios** [comment]
Thanks for the prompt edit, though that build didn't fix it. I did quickly check in the console if the new selectors worked and they did pick out both the unread and mute spans appropriately in my client.

**Comment by TheGoddessInari** [comment]
Unless I made a mistake, it's equivalent to the code that upstream uses since it's just a minor change. 

Here's an idea: Try their code in this build's `Custom Badge update JS` under advanced options (it overrides the shipped `unread_js`), try our `unread_js` code on their version.

It'll narrow down what's going on, probably.

**Comment by Athurios** [comment]
I also saw that the code is very similar, so I also had a hard time spotting why it doesn't work.

I tried both ideas you mentioned: Both ways broke the unread counter (no badge appearing at all). 
Though, looking at the code of both the shipped versions of `unread_js`, I think that has more to do with the fact that the lines `hamsket.updateBadge(count)` and `rambox.setUnreadCount(count)` respectively don't work with the other service.
I'll try messing around a bit with the `Custom Badge update JS` in the meantime.

**Comment by Athurios** [comment]
I managed to get it to work by using the upstream code for the `for` loop, while using Hamsket's selector and further code:
```
let checkUnread=()=>{const elements=document.querySelectorAll("#pane-side .P6z4j, .unread");let count=0;for (let i = 0; i < elements.length; i++) {if (elements[i].parentNode.parentNode.querySelectorAll('#pane-side *[data-icon="muted"]').length === 0) {count++;}}hamsket.updateBadge(count)};setInterval(checkUnread,1e3);let unregister_queue=[];navigator.serviceWorker.getRegistrations().then(registrations=>{for(const registration of registrations)unregister_queue.push(registration.unregister());return unregister_queue}).then(queue=>{}).catch(err=>{});
```
I've not done a pull request before, but if you're busy I can try my hand at it.

**Comment by TheGoddessInari** [comment]
Like #131, should be fixed in b5a13d2. Let me know if there are any further problems related to this specific breakage/fix.
